scripts/setup-hooks.sh

```
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# Ensure we are in the root directory
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)